### PR TITLE
Documentation: Add information regarding sandboxing, asar issues, improved code examples

### PR DESCRIPTION
**Pull request by tomm1996**
Add information about sandbox, asar issues and recommendations for environment variables to README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -227,6 +250,34 @@ The Electron service discussion forum is much less active than the WDIO one, but
...
+```json
+"package": "NODE_ENV=test electron-forge package"
+```
+
+```ts
+// forge.config.js
+module.exports = {
+    packagerConfig: {
+        asar: true,
+        prune: process.env.NODE_ENV === 'test',
```
Comment (christian-bromann) [inline README.md:275]: Shouldn't be it like is?
```suggestion
        prune: process.env.NODE_ENV !== 'test',
```

**Review by christian-bromann — COMMENTED** [review]
Thanks for adding these great docs, I have one comment but otherwise lgtm!

File: README.md [hunk 2]
```diff
@@ -92,11 +92,34 @@ export const config = {
 
 ### API Configuration
 
-If you wish to use the Electron APIs then you will need to import (or require) the preload and main scripts in your app. Somewhere near the top of your preload:
+If you wish to use the Electron APIs then you will need to import (or require) the preload and main scripts in your app. Somewhere near the top of your preload.
+To import node_modules in your `preload.js`, you have to disable the sandboxing in your `BrowserWindow` config.
```
Comment (erwinheitzman) [inline README.md:96]:
```suggestion
To import 3rd-party packages (node_modules) in your `preload.js`, you have to disable the sandboxing in your `BrowserWindow` config.
```
Can we add how to disable this?

File: README.md [hunk 3]
```diff
@@ -92,11 +92,34 @@ export const config = {
...
+
+It is recommended to not disable sandbox mode in Production, so it can be set with an Environment Variable from your package.json.
```
Comment (erwinheitzman) [inline README.md:98]:
```suggestion
It is not recommended to disable sandbox mode in production, to control this behaviour you can set the `NODE_ENV` environment variable.
```

File: README.md [hunk 4]
```diff
@@ -92,11 +92,34 @@ export const config = {
...
+      preload: path.join(__dirname, 'preload.js'),
+  }
+  // ...
+});
+```
+
+Load `wdio-electron-service/preload` conditionally in your `preload.js`
+
+```ts
+if (process && process.env.NODE_ENV === 'test') {
```
Comment (erwinheitzman) [inline README.md:121]: process refers to the current script, therefore I do not thing it can be falsy

**Review by erwinheitzman — COMMENTED** [review]
Thank you very much for the PR @tomm1996 👍 

I'll let Christian have a look as well since he originally wanted to pick this up

File: README.md (hunk 4 above)
Comment (tomm1996) [reply README.md:121]: @erwinheitzman I think it is undefined in preload.js if `sandbox` is set to true

**Review by christian-bromann — APPROVED** [review]
LGTM 👍 

@goosewobbler please take a look as well

**Review by erwinheitzman — APPROVED** [review]
LGTM too 👍

**Review by goosewobbler — APPROVED** [review]
Looks good, I tweaked the wording a bit.  I wasn't aware of this sandbox issue, my testing has worked fine without specifying `sandbox: false`.  I'll create another issue to update the example repos since it seems explicitly enabling sandboxing is [recommended](https://github.com/electron/electron/blob/main/docs/tutorial/sandbox.md#configuring-the-sandbox).

**Pull request merged**